request_test.cc

```diff
@@ -0,0 +1,138 @@
+#include "gtest/gtest.h"
+#include "request.h"
+#include <string>
+
+class RequestTest : public ::testing::Test {
+  protected:
+    std::string CreateRequestString(std::string method, std::string uri, 
```

Comment:
Can you line up the parameters. 
Ex:
std::string method,
std::string uri,
etc